plugins/meta/firewall/nftables.go

```diff
@@ -0,0 +1,350 @@
+// Copyright 2018 CNI authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/containernetworking/cni/pkg/types/current"
+	"net"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+type nftBackend struct {
+	targetTable     string
+	targetChain     string
+	targetHandle    uint64
+	targetInterface string
+	targetNetwork   *net.IPNet
+	rules           []*nftRule
+	markedDelRules  []uint64
+	newRules        [][]string
+}
+
+type nftRule struct {
+	text    string
+	handle  uint64
+	verdict string
+}
+
+func newNftRule(s string) (*nftRule, error) {
+	r := &nftRule{
+		text:    s,
+		verdict: "unknown",
+	}
+	if err := r.parseText(); err != nil {
+		return nil, err
+	}
+	if strings.HasSuffix(r.text, " accept") {
+		r.verdict = "accept"
+	}
+	if strings.HasSuffix(r.text, " drop") {
+		r.verdict = "drop"
+	}
+	return r, nil
+}
+
+func (r *nftRule) parseText() error {
+	offset := strings.LastIndex(r.text, "# handle ")
```

Comment:
`strings.Split` seems more efficient here

Reply:
@mars1024 , will change it to use `Split`.